packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -72,11 +77,24 @@ class BaseFileLink(BaseModel):
         examples=["MyFile.txt"],
     )
 
-    class Config:
-        extra = Extra.forbid
+    # class Config:
+    #    extra = Extra.forbid
 
 
 class SimCoreFileLink(BaseFileLink):
+    """ Input/Output is a reference to a dataset in the internal storage """
+
```

Comment:
I still do not understand that comment. it's a copy paste from the other one right?

Reply:
me neither... probably C&P. Cleaned up doc.